packaging/installer/UNINSTALL.md

```diff
@@ -33,8 +33,20 @@ NETDATA_ADDED_TO_GROUPS="<additional groups>"  # Additional groups for a user ru
 
 3.  Run `netdata-uninstaller.sh` as follows
 
+    3.1 Interactive mode (Default)
+
+The default mode in the uninstaller script is `interactive`. This means that the script provides to the user the option to reply with "yes" (`y`/`Y`) or no (`n`/`N`) to control the removal of each netdata asset in the filesystem.
+
 ```sh
 ${NETDATA_PREFIX}/usr/libexec/netdata/netdata-uninstaller.sh --yes --env <environment_file>
```

Comment:
Should be indented to be at the same level as 3.1

Reply:
Thanks @ilyam8! Fixed.